_layouts/empty.html

```diff
@@ -0,0 +1 @@
+{{ content }}
```

Comment:
This file seems a bit redundant - I *think* it's possible to not specify a layout in Jekyll, though I could be wrong